addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -212,9 +212,23 @@ $i: 1;
 
 .pac-container { // google map autosuggestion
     z-index: $zindex-modal-backdrop; // > $o-we-zindex
-    min-width: 400px;
+    min-width: 160px;
+    font-size: 12px;
+    border: $o-we-sidebar-content-field-dropdown-border-width solid $o-we-sidebar-content-field-dropdown-border-color;
+    background-color: $o-we-sidebar-content-field-dropdown-bg;
+    box-shadow: $o-we-sidebar-content-field-dropdown-shadow;
 
     &:after {
         display: none;
     }
+    .pac-item {
+        border: $o-we-sidebar-content-field-border-width solid $o-we-sidebar-content-field-border-color;
+        border-radius: $o-we-sidebar-content-field-border-radius;
+        background-color: $o-we-sidebar-content-field-clickable-bg;
+        color: $o-we-sidebar-content-field-clickable-color;
+        &:hover {
+            border: none;
```

Comment:
can you improve it to keep the same height when an item is active as it is not active ? removing the border reduces the height of the item when hovering it. thanks